### ISSUE TITLE
javafx-web for iOS: includes unused webkit and dat file

### ISSUE THREAD

**Issue by jperedadnr** [gluonhq/substrate#857]
The javafx-web jar for Mac is used for both desktop and iOS, containing the classes and resources for both cases.

While for desktop having the iOS files is just a very small overhead, for iOS, having the desktop files adds around of 30 MB of webkit, and what's more, because of the AOT build, includes a dat file from JDK/lib/security/public_suffix_list.dat which contains links to banned countries by EEUU, causing a rejection from the App Store.

Ideally, having a dedicated javafx-web-ios artifact without webkit will be better. In the meantime, we need a way to remove that .dat file. As an option, we could add `-H:ExcludeResources=public_suffix_list.dat` by default in Substrate, providing we bump the GraalVM version.

**Comment by johanvos** [comment]
The names should be the same (javafx-web) but we should indeed have separate modules for mac and ios.

**Comment by jperedadnr** [comment]
Indeed, we have separated modules. `javafx-web.jar` for iOS excludes most of WebKit for desktop classes, including `PublicSuffixes` that was calling `public_suffix_list.dat`.

However, there was a file `modules/javafx.web/src/main/resources/com/sun/webkit/network/effective_tld_names.dat` (with the same content) that was part of the jar for iOS, causing this issue .

After JDK-8254049, the file has been removed, and this issue has been solved.
